### PR TITLE
feat(composer-app): Markdown export & import

### PR DESCRIPTION
**Pull request by thure**
This PR enables export to and import from Markdown.

https://user-images.githubusercontent.com/855039/222934941-39e367e7-9880-49f8-bc43-71c7763e9dc4.mp4

### CODE REVIEW

File: packages/apps/composer-app/package.json [hunk 1]
```diff
@@ -24,15 +24,20 @@
...
     "@sentry/react": "^7.17.2",
+    "@types/showdown": "^2.0.0",
     "phosphor-react": "^1.4.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.4.0"
+    "react-drag-drop-files": "^2.3.8",
+    "react-router-dom": "^6.4.0",
+    "showdown": "^2.1.0",
+    "turndown": "^7.1.1"
```
Comment (thure) [inline packages/apps/composer-app/package.json:33]: `showdown` is capable of both import and export, however when exporting it appears to need non-trivial intervention in order to parse our styled HTML correctly, whereas `turndown` converts with only one trivial issue which I've resolved in this PR.

Comment (wittjosiah) [reply packages/apps/composer-app/package.json:33]: if you think it'd be better to use showdown for export long term perhaps leave a todo noting what the issue is

Comment (thure) [reply packages/apps/composer-app/package.json:33]: Done 👍

File: packages/apps/patterns/react-composer/src/components/Composer/Composer.tsx [hunk 2]
```diff
@@ -155,6 +153,25 @@ export const Composer = ({ document, field = 'content', placeholder, slots = {}
...
 
+const PureComposer = ({ editor, slots = {} }: PureComposerProps) => {
+  // Reference:
+  // https://tiptap.dev/installation/react
+  // https://github.com/ueberdosis/tiptap
+  // https://tiptap.dev/guide/output/#option-3-yjs
   return <EditorContent {...slots?.root} editor={editor} />;
 };
+
+const EditorComposer = (props: EditorProps) => {
```
[the remark this answers is not in the document]
Comment (thure) [reply packages/apps/patterns/react-composer/src/components/Composer/Composer.tsx]: I adjusted the names for better clarity, was getting a bit confused myself. Also added jsdocs to the internal components.

**Pull request merged**